Packaging.Targets/Packaging.Targets.csproj

```diff
@@ -22,6 +22,12 @@
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.5|AnyCPU'">
     <PlatformTarget>AnyCPU</PlatformTarget>
   </PropertyGroup>
+  <!-- Copy build task assemby to build directory -->
+  <PropertyGroup>
+    <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
```

Comment:
Did you plan to use this variable in the `PackagePath` values?

Reply:
It's a standard property that's needed for Packaging.Targets.dll to be placed into `tools` directory.

Reply:
makes sense, thanks.